ctapipe/coordinates/camera_frame.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import (
+    BaseCoordinateFrame,
+    CoordinateAttribute,
+    QuantityAttribute,
+    TimeAttribute,
+    EarthLocationAttribute,
+    FunctionTransform,
+    frame_transform_graph,
+    CartesianRepresentation,
+    UnitSphericalRepresentation,
+)
+
+from .horizon_frame import HorizonFrame
+from .telescope_frame import TelescopeFrame
+from .representation import PlanarRepresentation
+
+
+class CameraFrame(BaseCoordinateFrame):
+    '''Camera coordinate frame.  The camera frame is a simple physical
+    cartesian frame, describing the 2 dimensional position of objects
+    in the focal plane of the telescope Most Typically this will be
+    used to describe the positions of the pixels in the focal plane
+
+    Attributes
+    ----------
+
+    focal_length : u.Quantity[length]
+        Focal length of the telescope as a unit quantity (usually meters)
+    rotation : u.Quantity[angle]
+        Rotation angle of the camera (0 deg in most cases)
+    telescope_pointing : SkyCoord[HorizonFrame]
+        Pointing direction of the telescope as SkyCoord in HorizonFrame
+    obstime : Time
+        Observation time
+    location : EarthLocation
+        location of the telescope
+    '''
+    default_representation = PlanarRepresentation
+
+    focal_length = QuantityAttribute(default=0, unit=u.m)
+    rotation = QuantityAttribute(default=0 * u.deg, unit=u.rad)
+    telescope_pointing = CoordinateAttribute(frame=HorizonFrame, default=None)
+
+    obstime = TimeAttribute(default=None)
+    location = EarthLocationAttribute(default=None)
+
+
+@frame_transform_graph.transform(FunctionTransform, CameraFrame, TelescopeFrame)
+def camera_to_telescope(camera_coord, telescope_frame):
+    '''
+    Transformation between CameraFrame and TelescopeFrame.
+    Is called when a SkyCoord is transformed from CameraFrame into TelescopeFrame
+    '''
+    x_pos = camera_coord.cartesian.x
+    y_pos = camera_coord.cartesian.y
+
+    rot = camera_coord.rotation
+    if rot == 0:
```

Comment:
does it make sense to compare a float to exactly 0? This rotation angle may be computed elsewhere and subject to rounding error.  maybe `np.isclose()` should be used instead

Reply:
Actually, what is the purpose of this test?
If `rot==0`, don't you get anyway
```
x_rotated = x_pos
y_rotated = y_pos
```
?

Reply:
It's to not call sin and cos to spare some computing time

Reply:
I don't like it either, it was there before.

Reply:
Yeah this was to save computing time in the mist common case that no rotation is applies, feel free to get rid if you think it confuses the code. Ultimately the rotation shouldn't be needed, but for some cameras the camera coordinates passed needed to be rotated (this may even be fixed already).

Reply:
It works. In most cases the camera rotation is exactly zero, so no problems with the float comparison